DataStructures/Array/CyclicallyRotate.java

```diff
@@ -0,0 +1,34 @@
+public class CyclicallyRotate 
+{		
+		// Method for rotation
+		static void rotate(int arr[])
+		{

```

Comment:
Also I would move curly braces according to the oracle convention code.

From:
```
void method()
{
    // ...
}
```

To:
```
void method() {
    // ...
}
```

More: https://www.oracle.com/technetwork/java/codeconventions-150003.pdf